api/src/main/java/org/pmiops/workbench/workspaces/ManualWorkspaceMapper.java

```diff
@@ -6,22 +6,34 @@
 import java.util.List;
 import java.util.Map;
 import java.util.stream.Collectors;
+import javax.inject.Provider;
 import org.pmiops.workbench.api.Etags;
+import org.pmiops.workbench.config.WorkbenchConfig;
 import org.pmiops.workbench.db.model.DbStorageEnums;
 import org.pmiops.workbench.db.model.DbUser;
 import org.pmiops.workbench.db.model.DbUserRecentWorkspace;
 import org.pmiops.workbench.db.model.DbWorkspace;
 import org.pmiops.workbench.db.model.DbWorkspace.FirecloudWorkspaceId;
 import org.pmiops.workbench.firecloud.model.FirecloudWorkspace;
 import org.pmiops.workbench.firecloud.model.FirecloudWorkspaceAccessEntry;
+import org.pmiops.workbench.model.BillingStatus;
 import org.pmiops.workbench.model.RecentWorkspace;
 import org.pmiops.workbench.model.ResearchPurpose;
 import org.pmiops.workbench.model.SpecificPopulationEnum;
 import org.pmiops.workbench.model.UserRole;
 import org.pmiops.workbench.model.Workspace;
 import org.pmiops.workbench.model.WorkspaceAccessLevel;
+import org.springframework.stereotype.Service;
 
-public class WorkspaceConversionUtils {
+// We should migrate over to the Mapstruct supported WorkspaceMapper
+@Service
+public class ManualWorkspaceMapper {
```

Comment:
I made this injectable so I can pull in the WorkbenchConfig provider. 

Good example of why static utility classes can be hard to refactor.

I am guilty of writing this one heh.

Also, the name is intentionally a bit crude to draw attention to the need to migrate this class to the Mapstruct supported WorkspaceMapper.

Reply:
Done - https://precisionmedicineinitiative.atlassian.net/browse/RW-4367